docs/book/features/vsphere_csi_migration.md

```diff
@@ -181,10 +192,23 @@ To try out vSphere CSI migration in beta for vSphere plugin, perform the followi
                  Node ID:  k8s-node1
            Events:         <none>
 
-   - Repeat this for all workload nodes in the Kubernetes Cluster.
+   - Restart the kubelet on the control plane nodes using the command:
```

Comment:
`control plane` -> `workload`

Reply:
Is workload a new naming convention? I see control plane still used as the role in the kubectl get nodes.